server/server.js

```diff
@@ -11,6 +12,17 @@ app.use((req, res, next) => {
 // serve static assets
 app.use(express.static(path.join(__dirname, '../client/dist')));
 
+// outputs all comments for a song
+app.get('/api/songs/songId/comments', (req, res) => {
+  mongoDB.getComments(null, (err, data) => {
```

Comment:
what is the first argument, null, for?

Reply:
The first argument would be an error for getComments function in db.js. Specifying null as the first argument will retrieve the comments instead.